config/tasks/task_steps_config-recipe_m2_apply_tourniquet.yaml

```diff
@@ -0,0 +1,112 @@
+# Schema version.
+version: "1.0"
+
+# Reference to the activity classification labels configuration that we will
+# reference into.
+activity_labels: "./config/activity_labels/recipe_m2_apply_tourniquet.yaml"
+
+# Reference to the file defining the mean and standard deviation of the
+# activity classifications to be used by the HMM. For N activities, both the
+# mean and standard deviation should be N x N matrices such that when activity
+# i is actually occuring, the classifier will emit confidence
+# mean[i, j] +/- std[i, j] for activity j.
+activity_mean_and_std_file: "./model_files/recipe_m2_apply_tourniquet.npy"
+
+# Task title for display purposes.
+title: "M2 Apply tourniquet"
```

Comment:
This will need to be just "M2" for BBN integration reasons --> this string gets set to the TaskUpdate message `task_name` field which is used to name a correlated field in the BBN Update message. To my knowledge only "M2" intersects their expected task names.